# --- T2-COPYRIGHT-NOTE-BEGIN ---
# T2 SDE: package/*/hfsutils/hotfix-sys_errlist.patch
# Copyright (C) 2024 The T2 SDE Project
# 
# This Copyright note is generated by scripts/Create-CopyPatch,
# more information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License version 2 as used by the T2 SDE.
# --- T2-COPYRIGHT-NOTE-END ---

strerror.c:34:14: error: conflicting types for 'sys_errlist'; have 'char *[]'
   34 | extern char *sys_errlist[];
      |              ^~~~~~~~~~~
In file included from /srv/t2/build/sparc-24.12-generic-sparc-v8-cross-linux/usr/include/stdio.h:853,
                 from strerror.c:26:
/srv/t2/build/sparc-24.12-generic-sparc-v8-cross-linux/usr/include/bits/sys_errlist.h:27:26: note: previous declaration of 'sys_errlist' with type 'const char * const[]'
   27 | extern const char *const sys_errlist[];

--- hfsutils-3.2.6/strerror.c.vanilla	2024-12-05 19:17:57.986475237 +0100
+++ hfsutils-3.2.6/strerror.c	2024-12-05 19:19:15.729808071 +0100
@@ -31,7 +31,7 @@
 
 # ifdef HAVE_SYS_ERRLIST
 
-extern char *sys_errlist[];
+//extern const char *sys_errlist[];
 extern int sys_nerr;
 
 # endif
